domains/link.yxz.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "yz9551",
+        "email": "",
+        "discord": "yz9551"
```

Comment:
```suggestion
        "username": "yz9551",
        "discord": "yz9551"
```

Reply:
I read on previous versions of the readme to set an email but leave it blank, if its no longer needed, I guess it can be removed.